src/components/Books/Books.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { SFC, useEffect, useState } from 'react';
+import IBook from '../../models/IBook';
+import { fetchBooks } from './api';
+import View from './BooksView';
+
+interface IBooksState {
+    books: IBook[];
+}
+
+const Books: SFC = () => {
+  const [books, loadBooks] = useState<IBook[]>([]);
+  useEffect(() => {
```

Comment:
This actually works without wrapping the api call in `useEffect`, but that still seems to be the recommended way to handle this. From [the docs](https://reactjs.org/docs/hooks-effect.html):

> Tip
> If you’re familiar with React class lifecycle methods, you can think of useEffect Hook as componentDidMount, componentDidUpdate, and componentWillUnmount combined.